pandas/tests/series/test_api.py

```diff
@@ -10,7 +10,7 @@
 from pandas import Index, Series, DataFrame, date_range
 from pandas.core.indexes.datetimes import Timestamp
 
-from pandas.compat import range
+from pandas.compat import range, lzip, isidentifier, string_types
```

Comment:
can you add a test that has 101 columns and assert first 100 there and last 1 is not.

Reply:
can you do thsi

Reply:
I've put this in the other test (See [here](https://github.com/pandas-dev/pandas/pull/16326/files#diff-db91df05b120c5eb39d90899b54b5321R266) and [there](https://github.com/pandas-dev/pandas/pull/16326/files#diff-db91df05b120c5eb39d90899b54b5321R277))
Please tell me if you prefer that I put it in a separate test.